packages/fx-core/src/common/m365/packageService.ts

```diff
@@ -100,12 +102,12 @@ export class PackageService {
       } while (true);
     } catch (error: any) {
       this.logger?.error("Sideloading failed.");
-      if (error.response) {
-        this.logger?.error(JSON.stringify(error.response.data));
-        this.traceError(error);
-      } else {
-        this.logger?.error(error.message);
-      }
+      // if (error.response) {
```

Comment:
What's the purpose to change the error handling in this file?
Conditional log/trace error is required for m365 action in different runtime, vsc v.s. cli.